content/F2016/F16-event-gooutside.md

```diff
@@ -0,0 +1,20 @@
+title: WiCS and CSC Go Outside
+Date: 2016-10-13 18:00
```

Comment:
this date is actually when you write the post, and not when the event is (pretty sure)